bandyer-android-design/src/main/res/values/themes_call.xml

```diff
@@ -30,6 +30,7 @@
         <attr name="bandyer_userStatusStyle" format="reference"/>
 
         <attr name="bandyer_whiteboardDialogStyle" format="reference" />
+        <attr name="bandyer_screenShareDialogStyle" format="reference" />
```

Comment:
This attribute is not defined in the call theme (you should follow the whiteboard dialog example)